pagefaultgames/pokerogue
# [Localization] Updated and localized "The stack for this item is full." message

## What are the changes?

Localized the "The stack for this item is full." message and changed it so it says which item has reached its stack limit.

## Why am I doing these changes?



The text was hardcoded and (thanks to @EnochG1) sometimes, e.g., at the start of [this daily run](https://www.youtube.com/watch?v=Ly4tODMs15w), you don't know which item has reached its stack limit.
## What did change?


Added the "itemStackFull" locale in battle.ts and changed the text to display the replaced item as well.
### Screenshots/Videos


### Before (en)
![image](https://github.com/user-attachments/assets/f4a2ae39-8e12-494a-8eca-6b97d3adbb38)

### After (en)
![image](https://github.com/user-attachments/assets/38be4073-d0fc-415e-a98e-8792d55d9ede)

### Before (pt_BR)
![image](https://github.com/user-attachments/assets/49ebcacb-3f39-47aa-a718-145c8963f590)

### After (pt_BR)
![image](https://github.com/user-attachments/assets/09a5cabf-9f77-4b00-8a68-5fd2b244dba1)

## How to test the changes?




Change these overrides:
```
readonly STARTING_WAVE_OVERRIDE: integer = 50;
readonly STARTING_LEVEL_OVERRIDE: integer = 100;
readonly STARTING_MODIFIER_OVERRIDE: Array<ModifierOverride> = [{ name: "GOLDEN_POKEBALL", count: 3 }];
```
Win the battle and check the message

## Checklist
